### PR TITLE
Update README with min versions for tools

### PR DESCRIPTION
**Pull request by Nana-EC**
**Detailed description**:
Questions around min versions for tools occasionally come up, we should explicitly note it

- Update README.md with minimum versions of used tools

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:
Feel free to recommend any extras

**Checklist**

- [x] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,22 +79,26 @@ to understanding the mirror node operations.
...
 
-Ensure these tools are installed prior to running the mirror node
+Ensure these tools are installed (note minimum versions) prior to running the mirror node
 
-- [Docker](https://www.docker.com/products/docker-desktop)
+- [Docker](https://www.docker.com/products/docker-desktop) - v3.3.3 (minimum)
+- [NodeJs](https://nodejs.org) - v14.16.0 (minimum)
+- [Npm](https://www.npmjs.com/get-npm) - v6.14.11 (minimum)
+- [Go](https://golang.org/dl/) - v1.13+ (minimum)
+- Java - JDK 11 (minimum), [OpenJDK 11](https://openjdk.java.net/projects/jdk/11/) (recommended)
```
Comment (steven-sheehy) [inline README.md:101]: We actually use AdoptOpenJDK everywhere and should recommend that.

Comment (steven-sheehy) [inline README.md:101]: Specifying JDK 11 and OpenJDK 11 is redundant.

File: README.md [hunk 2]
```diff
@@ -79,22 +79,26 @@ to understanding the mirror node operations.
...
 ### Prerequisite Tools
 
-Ensure these tools are installed prior to running the mirror node
+Ensure these tools are installed (note minimum versions) prior to running the mirror node
 
-- [Docker](https://www.docker.com/products/docker-desktop)
+- [Docker](https://www.docker.com/products/docker-desktop) - v3.3.3 (minimum)
+- [NodeJs](https://nodejs.org) - v14.16.0 (minimum)
+- [Npm](https://www.npmjs.com/get-npm) - v6.14.11 (minimum)
+- [Go](https://golang.org/dl/) - v1.13+ (minimum)
```
Comment (steven-sheehy) [inline README.md:100]: Specifying `v1.13+` and `(minimum)` is redundant. Recommend `(minimum)` be removed from all and just add the plus to everything.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (xin-hedera) [reply README.md:100]:
```suggestion
- [Go](https://golang.org/dl/) - v1.16+
```

File: README.md (hunk 1 above)
Comment (Nana-EC) [reply README.md:101]: Ahh then we should update the Installation.md also.
Will do that

File: README.md (hunk 2 above)
Comment (Nana-EC) [reply README.md:100]: Works for me

**Review by SimiHunjan — DISMISSED** [review]
👍

File: README.md (hunk 1 above)
Comment (Nana-EC) [reply README.md]: Updated

Comment (Nana-EC) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (Nana-EC) [reply README.md]: Updated

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by ijungmann — APPROVED** [review]

**Pull request merged**